server/views/components/meta-unit/meta-unit.njk

```diff
@@ -9,22 +9,18 @@
     <p class="{{ {s:'HNL5' , m:'HNL4'} | fontClasses }} {{ {s:2} | spacingClasses({margin: ['bottom']}) }}">{{model.content | safe}}</p>
   {% else %}
     {% for item in model.content %}
-      {% if item.datetime %}
+      {% if item | isString %}
+        <p class="{{ {s:'HNL5' , m:'HNL4'} | fontClasses }} {{ {s:2} | spacingClasses({margin: ['bottom']}) }}">
+          {{item}}
+        </p>
+      {% else %}
         {% if loop.first %}
-          <p class="{{ {s:'HNL5' , m:'HNL4'} | fontClasses }} {{ {s:2} | spacingClasses({margin: ['bottom']}) }}">
+        <ul class="plain-list {{ {s:2} | spacingClasses({margin: ['bottom']}) }} {{ {s:0} | spacingClasses({margin: ['top', 'left', 'right'], padding: ['top', 'left', 'right']}) }}">
```

Comment:
Could we not just put this on line 10 and avoid the conditional?

Reply:
could end up wrapping a <p> in a <ul> then